src/event_loop.rs

```diff
@@ -486,7 +492,7 @@ unsafe impl rwh_05::HasRawDisplayHandle for ActiveEventLoop {
 ///
 /// - A zero-sized type that is likely optimized out.
 /// - A reference-counted pointer to the underlying type.
-#[derive(Clone)]
+#[derive(Clone, PartialEq, Eq)]
```

Comment:
What's the user story for this one?

Reply:
Ha!
I would honestly not be able to come up with one.

Happy to remove it if you don't like it!